src/AllCountries.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from "react";
+import Card from "./Card";
+import SidebarWidth from "./Sidebar";
+import { filterOptions } from "./ConstantData";
+import Filter from "./Filter";
+
+export default function AllCountries() {
+  const FetchCountriesData = async () => {
```

Comment:
Bro, camelCase for function names as well. We capitalize only the Components. For functions, we use camelCase